eox_tenant/tenant_wise/proxies.py

```diff
@@ -176,12 +176,20 @@ def get_queryset(self):
         Call parent method and filter the certificates by org.
         """
         generated_certificates = super(TenantCertificateManager, self).get_queryset()
-        excluded_courses = [
-            certificate.course_id
-            for certificate in generated_certificates
-            if certificate.course_id.org not in getattr(settings, "course_org_filter", [])
-        ]
-        return generated_certificates.exclude(course_id__in=excluded_courses)
+        org_filter = getattr(settings, "course_org_filter", [])
+
+        if isinstance(org_filter, six.string_types):
+            org_filter = set([org_filter])
+
+        if org_filter:
+            q_objects = models.Q()  # Create an empty Q object to start with.
```

Comment:
In my opinion the line is clear enough without the comment, but it does not hurt either.